app/forms/recaptcha_form.rb

```diff
@@ -107,6 +109,10 @@ def is_result_error?(error_code)
     RESULT_ERRORS.include?(error_code)
   end
 
+  def result_reason_exempt(result)
+    EXEMPT_RESULT_REASONS.any? { |reason| result.reasons.include?(reason) }
```

Comment:
No strong preference from me, but another way we could do this is checking if an intersection of the two arrays is blank or not

```suggestion
    (EXEMPT_RESULT_REASONS & result.reasons).any?
```

https://ruby-doc.org/core-3.1.1/Array.html#method-i-26